webClient/src/app/core/menu-bar/menu-bar.config.ts

```diff
@@ -142,7 +142,8 @@ export const MENU = [
               action: {
                 internalName: 'createFile'
               },
-              keyMap: '' // [Ctrl+N] won't work due to browser conflicts. Keybindings will need
+              keyMap: '', // [Ctrl+N] won't work due to browser conflicts. Keybindings will need
```

Comment:
You can remove the [Ctrl + N] comment, no longer relevant. A new comment up at the top would be good, something like "Cross browser research shows ALT + [ ] are the least conflicting combinations, so we use these"

Reply:
removed old comments, also moved shortcut attribute to existing attribute keymap